cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -19,9 +19,9 @@ spec:
     # v13 is mimic, v14 is nautilus, and v15 is octopus.
     # RECOMMENDATION: In production, use a specific version tag instead of the general v14 flag, which pulls the latest release and could result in different
     # versions running within the cluster. See tags available at https://hub.docker.com/r/ceph/ceph/tags/.
-    # If you want to be more precise, you can always use a timestamp tag such ceph/ceph:v16.2.4-20210514
+    # If you want to be more precise, you can always use a timestamp tag such ceph/ceph:v16.2.5-20210514
```

Comment:
```suggestion
    # If you want to be more precise, you can always use a timestamp tag such ceph/ceph:v16.2.5-20210708
```

Reply:
What about this one?

Reply:
ugh, ok should have everything in the next push...